addons/packages/metrics-server/0.5.1/bundle/config/overlays/update_overlay.yaml

```diff
@@ -0,0 +1,31 @@
+#@ load("@ytt:overlay", "overlay")
```

Comment:
Looking at all of these, is `update_overlay.yaml` a descriptive enough name that we will remember why these files exist months from now?  I'd probably prefer it be more descriptive.

Can we encode the "why" in the file name?